websites/docs/src/content/docs/en/getting_started/installation.mdx

```diff
@@ -33,18 +33,23 @@ Before you can install and use Ballerine, you will need the following:
 <PackageManagersTabs code={`run monorepo:init`}/>
 
 
-## Run one of the examples, or follow a guide from the [guide sections](https://docs.ballerine.com/en/learn/simple_kyb_guide/)
+### Installation steps on localhost with Docker
```

Comment:
Let's find a pattern for the titles we can repeat in a consistent manner. 🙂 I'm also familiar with just "Docker Usage"
```suggestion
### Local Development With Docker Steps
```